### ISSUE TITLE
Material radio button issues in screen readers

### ISSUE THREAD

**Issue by marcysutton** [angular/material#310]
To be made accessible, `aria-activedescendant` should be used on `material-radio-group` communicating the ID of the currently selected radio button. This attribute makes the control operable in NVDA without needing `role="application"` on a wrapping `<div>`.

Known issue with `role="radiogroup"` in JAWS: forms mode must be manually entered by the screen reader user. http://accessibleculture.org/articles/2012/09/aria-widgets-and-focus-forms-mode-support/
